Userland/Libraries/LibWebView/DOMTreeModel.h

```diff
@@ -51,8 +51,8 @@ class DOMTreeModel final : public GUI::Model {
 
     ALWAYS_INLINE static JsonArray const* get_children(JsonObject const& o)
     {
-        if (auto const* maybe_children = o.get_ptr("children"sv); maybe_children)
-            return &maybe_children->as_array();
+        if (auto maybe_children = o.get_array("children"sv); maybe_children.has_value())
+            return &maybe_children.value();
```

Comment:
Feel a little weird to change `AccessibilityTreeModel::get_children` to return a `Optional<JsonArray const&>` but not this one

Reply:
Sure, I'll change that too.